### PR TITLE
Adding Irish LC Workshop

### PR DESCRIPTION
**Pull request by davidfkane**
2 day workshop in August to allow at least some of our newly-minted instructors to do some training.

### CODE REVIEW

File: _data/workshops.yml [hunk 1]
```diff
@@ -1,4 +1,21 @@
 workshops:
+-
+    who: TBA
+    help: TBA
```
Comment (katrinleinweber) [inline _data/workshops.yml:4]: https://lirgroup.heanet.ie/index.php/2019/03/04/liber-lc-instructortraining2019/ has a few names. If they shall _not_ be added here this PR is good to merge, IMHO

Comment (libcce) [reply _data/workshops.yml:4]: Good catch @katrinleinweber! @davidfkane can you add the names of the instructors and helpers? Also, do you have the workshop site up and running so we can link to it?

**Comment by maneesha** [comment]
@libcce  Can we merge this?

**Comment by libcce** [comment]
We were waiting on more info about the workshop from @davidfkane.

@davidfkane did you find out who the instructors and helpers will be?

**Comment by davidfkane** [comment]
@libcce @katrinleinweber - We cannot say for sure who will be available to do the training at the moment.  If the names are required, we should hold off or cancel this merge for now.  We shall run the workshop but names not available at this time.  Best, David.

**Comment by katrinleinweber** [comment]
I [already wrote](https://github.com/LibraryCarpentry/librarycarpentry.github.io/pull/55#discussion_r297162843) that IMHO, this can be merged "if the[ names] shall not be added here" ;-) The workshop listing on the LC site is a useful announcement channel of the workshop itself, so date, location & topics are the important and hopefully final details.

**Comment by libcce** [comment]
@davidfkane Can you add your name in 'who' for the moment and then I can merge this. I agree with @katrinleinweber that it is a great place to advertise but it would be good to have a name associated with it just in case.

**Comment by libcce** [comment]
Hi @davidfkane I merged this PR and added you as the lead instructor for now.

**Pull request merged**